### PR TITLE
Fix ups

### PR DESCRIPTION
**Pull request by almarklein**
Fix some leftover references to up. See #482.

* [x] Fixed some docstrings that still mentioned minus/reverse from when `reference_up`  was called `gravity`.
* [x] Fixed that `ob.world.reference_up`  was initially a tuple (its an array everywhere else).
* [x] Fixed that setting `ob.world.reference_up` was undone when the local transform changed (the change in `world.reference_up` was not propagated to the `local` transform.
* [x] Added a test to make sure the above stays working, based using `camera.show_pos`.
* [x] Prefixed two callback functions with an underscore.  
* [x] Update all (I think) cases that still used `ob.up` instead of `ob.world.reference_up`.

### CODE REVIEW

File: pygfx/utils/transform.py [hunk 1]
```diff
@@ -590,30 +590,38 @@ def __init__(
...
-        self.own._reference_up = own_ref - parent_origin
-
+    def _parent_updated(self, parent: AffineBase):
+        self._propagate_reference_up()
         self.flag_update()
 
+    def _propagate_reference_up(self):
+        new_ref = la.vec_transform(self.reference_up, self.parent.inverse_matrix)
+        origin = la.vec_transform((0, 0, 0), self.parent.inverse_matrix)
+        self.own._reference_up = new_ref - origin
```
Comment (Korijn) [inline pygfx/utils/transform.py:612]: does this resulting vector potentially need normalizing?

File: tests/cameras/test_cameras.py [hunk 2]
```diff
@@ -79,6 +79,35 @@ def test_camera_show_methods():
...
+    camera.show_pos((0, 1, 0), up=(1, 0, 0))
+    assert tuple(camera.world.reference_up) == (1, 0, 0)
+
+    # Not specifying up, keeps the current up (no revert to default)
+    camera.show_pos((0, 1, 1))
+    assert tuple(camera.world.reference_up) == (1, 0, 0)
+
+    # Back to +Y
+    camera.show_pos((1, 0, 0), up=(0, 1, 0))
+    assert tuple(camera.world.reference_up) == (0, 1, 0)
```
Comment (Korijn) [inline tests/cameras/test_cameras.py:108]: this test relies on the fact that show_pos sets up implicitly, I would recommend to also test by setting camera.world.reference_up directly.

File: pygfx/utils/transform.py (hunk 1 above)
Comment (almarklein) [reply pygfx/utils/transform.py:612]: Good point.

Comment (almarklein) [reply pygfx/utils/transform.py]: Done!

File: tests/cameras/test_cameras.py (hunk 2 above)
Comment (almarklein) [reply tests/cameras/test_cameras.py:108]: Done!

**Review by Korijn — APPROVED** [review]

**Pull request merged**